automation/DataAggregator/schema.sql

```diff
@@ -43,7 +44,7 @@ CREATE TABLE IF NOT EXISTS flash_cookies (
 
 /*
 # crawl_history
- */
+*/
```

Comment:
Why these changes? The asterisks being on the same column is intentional, no?

Reply:
Yes this indenting was intentional